frontend/src/components/Table/ColumnSort.tsx

```diff
@@ -19,17 +19,17 @@ const Wrapper = styled('div')`
 `;
 
 interface Props {
-  active?: boolean;
```

Comment:
passing this prop was resulting in the active prop being set directly on the dom <svg> element which is invalid. using $ informs styled components that this is a transient prop, which will can be used like a regular prop but will not be written to the dom.